doc/user-guide.md

```diff
@@ -63,6 +63,17 @@ By default this will be the namespace that the operator is running in. To watch
 mgr, err := manager.New(cfg, manager.Options{Namespace: ""})
 ```
 
+It is also possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] to watch a specific set of namespaces:
+```Go
+var namespaces []string // List of Namespaces
+// Create a new Cmd to provide shared dependencies and start components
+mgr, err := manager.New(cfg, manager.Options{
+   NewCache: cache.MultiNamespacedCacheBuilder(namespaces),
+   MapperProvider:     restmapper.NewDynamicRESTMapper,
```

Comment:
Now that https://github.com/kubernetes-sigs/controller-runtime/pull/554 is merged and we've updated to v0.4.0, we should remove our dynamic rest mapper implementation and any references to it.

That can be done in a follow up though.